src/pyhf/infer/calculators.py

```diff
@@ -98,7 +99,7 @@ def __init__(self, data, pdf, init_pars=None, par_bounds=None, qtilde=False):
             pdf (~pyhf.pdf.Model): The statistical model adhering to the schema ``model.json``.
             init_pars (`tensor`): The initial parameter values to be used for fitting.
             par_bounds (`tensor`): The parameter value bounds to be used for fitting.
-            qtilde (`bool`): Whether to use qtilde as the test statistic.
+            qtilde (Bool): When ``True`` perform the calculation using the alternative test statistic, :math:`\\tilde{q}`, as defined in Equation (62) of :xref:`arXiv:1007.1727`.
```

Comment:
```suggestion
            qtilde (`bool`): When ``True`` perform the calculation using the alternative test statistic, :math:`\\tilde{q}`, as defined in Equation (62) of :xref:`arXiv:1007.1727`.
```

I'd keep it as the same style as

```python
>>> print(type(True))
<class 'bool'>
```

If we change to using Bool then we should make sure this gets changed everywhere and not just here.